build.sbt

```diff
@@ -7,16 +7,15 @@ import sbtassembly.MergeStrategy
 val projectVersion = "1.0-latest"
 
 organization := "com.gu"
-scalaVersion in ThisBuild := "2.12.11"
+scalaVersion in ThisBuild := "2.13.2"
 
 val compilerOptions = Seq(
   "-deprecation",
   "-Xfatal-warnings",
   "-feature",
   "-language:postfixOps",
   "-language:implicitConversions",
-  "-language:higherKinds",
-  "-Ypartial-unification"
+  "-language:higherKinds"
```

Comment:
Is this flag needed [since `2.13.1`](https://www.scala-lang.org/api/current/scala/languageFeature$$higherKinds.html)?

Reply:
Good shout!